### PR TITLE
F t31116 add routing props from demosplan UI

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://yaits.demos-deutschland.de/T31116

- no longer use routing in demosplan-ui
- instead of routing value, using props, and add this value in demosplan-core.

### CODE REVIEW

File: client/js/components/procedure/SegmentsBulkEdit/SegmentsBulkEdit.vue [hunk 1]
```diff
@@ -400,7 +401,7 @@ export default {
         params.assigneeId = this.actions.assignSegment.selected.id
       }
 
-      dpRpc('segment.bulk.edit', params)
+      dpRpc('segment.bulk.edit', params, 'rpc_generic_post')
```
Comment (salisdemos) [inline client/js/components/procedure/SegmentsBulkEdit/SegmentsBulkEdit.vue:404]: why is the 'rpc_generic_post' not part of the params?
here for the general Question

**Review by salisdemos — CHANGES_REQUESTED** [review]

File: client/js/components/procedure/SegmentsBulkEdit/SegmentsBulkEdit.vue (hunk 1 above)
Comment (ahmad-demos) [reply client/js/components/procedure/SegmentsBulkEdit/SegmentsBulkEdit.vue]: Revert here: https://github.com/demos-europe/demosplan-core/pull/718/commits/13bee5e02e09e441de4afd53ee38456e77bfe7c3

**Review by hwiem — DISMISSED** [review]
lgtm, code review only

**Review by salisdemos — APPROVED** [review]
Jsut nitpicks. No blocker

File: client/js/components/procedure/DpSimplifiedNewStatementForm.vue [hunk 2]
```diff
@@ -352,8 +355,7 @@
 </template>
 
 <script>
-import { DpInput, DpLabel } from '@demos-europe/demosplan-ui'
-import { DpAccordion, DpButtonRow, DpDatepicker, DpMultiselect, DpRadio, DpSelect, DpTextArea, DpUploadFiles } from '@demos-europe/demosplan-ui'
+import { DpAccordion, DpButtonRow, DpDatepicker, DpInput, DpLabel, DpMultiselect, DpRadio, DpSelect, DpTextArea, DpUploadFiles } from '@demos-europe/demosplan-ui'
```
[the remark this answers is not in the document]
Comment (ahmad-demos) [reply client/js/components/procedure/DpSimplifiedNewStatementForm.vue]: Done in https://github.com/demos-europe/demosplan-core/pull/718/commits/90ccf875c81add38d4944b9c58aeb4880e8e9114

**Review by salisdemos — APPROVED** [review]

**Review by spiess-demos — CHANGES_REQUESTED** [review]
Looks good, however i'd suggest to also bump demosplan-ui to a compatible version within this PR.

File: client/js/components/procedure/SegmentsBulkEdit/SegmentsBulkEdit.vue [hunk 3]
```diff
@@ -123,6 +123,9 @@
               fullscreenButton: true,
               linkButton: true
             }"
+            :routes="{
+              boilerplateEditViewRoute: Routing.generate('DemosPlan_procedure_boilerplate_list', { procedure: procedureId })
+            }"
```
Comment (gruenbergerdemos) [inline client/js/components/procedure/SegmentsBulkEdit/SegmentsBulkEdit.vue:128]: Here eg.

**Review by gruenbergerdemos — DISMISSED** [review]
In https://github.com/demos-europe/demosplan-ui/pull/104 DpRecommendationModal and DpBoilerPlateModal are moved to core and therefore we don't need any routing changes there. So in this PR all occurrences for this two components (boilerplateEditViewRoute and similarRecommendationsRoute) need to be reverted.

**Review by spiess-demos — APPROVED** [review]
Thanks for adding the version bump! Lgtm!

**Pull request merged**